File: vb - partial complete/BAIS3150CodeSampleSystem/App_Code/Student.vb
Imports Microsoft.VisualBasic

Public Class Student
    Private mStudentID As String
    Public Property StudentID() As String
        Get
            Return mStudentID
        End Get
        Set(ByVal value As String)
            mStudentID = value
        End Set
    End Property

    Private mFirstName As String
    Public Property FirstName() As String
        Get
            Return mFirstName
        End Get
        Set(ByVal value As String)
            mFirstName = value
        End Set
    End Property

    Private mLastName As String
    Public Property LastName() As String
        Get
            Return mLastName
        End Get
        Set(ByVal value As String)
            mLastName = value
        End Set
    End Property

    Private mEmail As String
    Public Property Email() As String
        Get
            Return mEmail
        End Get
        Set(ByVal value As String)
            mEmail = value
        End Set
    End Property



End Class

File: c# - fully complete/CSBAIS3150CodeSampleSystem/BAIS3150CodeSampleSystem/App_Code/Student.cs
using System;
using System.Collections.Generic;
using System.Web;

/// <summary>
/// Summary description for Student
/// </summary>
public class Student
{
    private string mStudentId;
    public string StudentId
    {
        get { return mStudentId; }
        set { mStudentId = value; }
    }

    private string mFirstName;
    public string FirstName
    {
        get { return mFirstName; }
        set { mFirstName = value; }
    }

    private string mLastName;
    public string LastName
    {
        get { return mLastName; }
        set { mLastName = value; }
    }

    private string mEmail;
    public string Email
    {
        get { return mEmail; }
        set { mEmail = value; }
    }


    public Student()
    {
        //
        // TODO: Add constructor logic here
        //
    }
}